<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketGraphics</class>
 <widget class="QWidget" name="RocketGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout" stretch="3,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="bothLayout" stretch="1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QFrame" name="mapFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame
{
	border: 5px solid rgb(212,235,242);
	background-color: white;
	border-radius: 25px;
} 


</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QQuickWidget" name="map">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="resizeMode">
             <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="altitudeFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame
{
	border: 5px solid rgb(212,235,242);
border-radius: 25px;
	background-color: white;
	
} 




</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QStackedWidget" name="altitudeStack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QCustomPlot" name="altitudeGraph"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="printFrame">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QFrame" name="serialFrame">
         <property name="styleSheet">
          <string notr="true">#serialFrame
{
	border: 5px solid rgb(212,235,242);
	background-color: white;
	border-radius: 25px;
} 

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QListView" name="serialPrint">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>11</pointsize>
              <italic>false</italic>
              <bold>true</bold>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame
{
	border: 0px;
	background-color: white;
} 

</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="orderFrame">
         <property name="styleSheet">
          <string notr="true">#orderFrame
{
	border: 5px solid rgb(212,235,242);
	background-color: white;
	border-radius: 25px;
} 

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QListView" name="orderPrint">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>11</pointsize>
              <italic>false</italic>
              <bold>true</bold>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame
{
	border: 0px;
	background-color: white;
} 

</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>teston()</slot>
  <slot>ledon()</slot>
  <slot>ledoff()</slot>
  <slot>ignitionon()</slot>
  <slot>ignitionoff()</slot>
  <slot>paracason()</slot>
  <slot>paracasoff()</slot>
  <slot>alarmaon()</slot>
  <slot>alarmaoff()</slot>
  <slot>testoff()</slot>
  <slot>exporttest()</slot>
  <slot>exportgraph()</slot>
 </slots>
</ui>
